ossl/src/lib.rs

```diff
@@ -219,6 +220,29 @@ impl OsslContext {
         OsslContext { context: ctx }
     }
 
+    pub fn load_configuration_file(
+        self,
```

Comment:
having another look here. I think we should have get here the `&self` as we do not want to take the ownership here nor in the case of `load_default_configuration()`. I missed this during the first review. WDYT?